packages/manager/apps/pci-private-registry/src/pages/create/steps/RegionStep.tsx

```diff
@@ -90,63 +139,81 @@ export default function RegionStep({
-        !isLocked && {
-          action: () => {
-            stepsHandle.close(StepEnum.NAME);
-            stepsHandle.uncheck(StepEnum.NAME);
-            stepsHandle.unlock(StepEnum.NAME);
-
-            stepsHandle.close(StepEnum.PLAN);
-            stepsHandle.uncheck(StepEnum.PLAN);
-            stepsHandle.unlock(StepEnum.PLAN);
-
-            stepsHandle.uncheck(StepEnum.REGION);
-            stepsHandle.unlock(StepEnum.REGION);
-          },
-          label: tCommonField('common_stepper_modify_this_step'),
-        }
-      }
+      edit={{
+        action: () => {
+          stepsHandle.close(StepEnum.NAME);
+          stepsHandle.uncheck(StepEnum.NAME);
+          stepsHandle.unlock(StepEnum.NAME);
+
+          stepsHandle.close(StepEnum.PLAN);
+          stepsHandle.uncheck(StepEnum.PLAN);
+          stepsHandle.unlock(StepEnum.PLAN);
+
+          stepsHandle.uncheck(StepEnum.REGION);
+          stepsHandle.unlock(StepEnum.REGION);
+        },
+        label: t('common_field:common_stepper_modify_this_step'),
+      }}
     >
-      {isPending && (
+      {isPending ? (
         <OsdsSpinner
           inline
           size={ODS_SPINNER_SIZE.md}
           className="block text-center"
         />
-      )}
-      {!isPending && (
-        <TilesInputComponent
-          items={regions}
-          value={store.state.region}
-          onInput={(region) => {
-            store.set.region(region);
-          }}
-          label={(region) => region.microLabel}
-          group={{
-            by: (region) => region.continentLabel,
-            label: (continent: string) => (
-              <OsdsText
-                break-spaces="false"
-                size={ODS_TEXT_SIZE._600}
-                color={ODS_THEME_COLOR_INTENT.text}
-                level={ODS_THEME_TYPOGRAPHY_LEVEL.body}
-                hue={ODS_THEME_COLOR_HUE._400}
-              >
-                <div className="whitespace-nowrap px-2 text-lg">
-                  {continent ||
-                    (localisations?.continents || []).find(
-                      (c) => c.code === 'WORLD',
-                    )?.name}
-                </div>
-              </OsdsText>
-            ),
-            showAllTab: true,
-          }}
-        />
+      ) : (
+        <div>
+          {!isFeatureAvailabilityPending && is3AZEnabled && (
+            <div>
+              <DeploymentTilesInput
+                name="deployment"
+                value={selectedRegionGroup}
+                onChange={handleRegionChange}
+                deployments={deploymentZones}
+                locked={isLocked}
+              />
+              <div className="mb-6">
+                <Subtitle>
+                  {t('create:private_registry_create_location')}
+                </Subtitle>
+              </div>
+            </div>
+          )}
+
+          {isLocked ? (
+            <RegionSummary region={store.state.region} />
+          ) : (
+            <TilesInputComponent
```

Comment:
Why not use the new TilesInput from pci-common ?

Reply:
yes at the start of the feature we talked together about using this component, but in the end (if I remember correctly) it would have meant big changes, so it was decided to stick with the MRC component.